apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -423,11 +500,17 @@ export default function PublishWizard({
           if (typeof folderId === 'string') {
             setPath(folderId);
           }
+
           setIsChangeFolderModalOpened(false);
         }}
         type={type}
         depth={depth}
-        rootFolderId={getFileRootId()}
+        rootFolderId={
+          type === SharingType.Conversation ||
+          type === SharingType.ConversationFolder
+            ? constructPath(ApiKeys.Conversations, 'public')
+            : constructPath(ApiKeys.Prompts, 'public')
```

Comment:
Avoid to use `ApiKeys` in our logic and use existing util-methods istead of own construction of paths
```suggestion
            ? getRootId({ featureTupe: FeatureType.Chat , bucket: 'public')
            : getRootId({ featureTupe: FeatureType.Prompt , bucket: 'public')
```